packages/create-pantheon-decoupled-kit/src/templates/partials/nextjs-shared/layoutT.ts

```diff
@@ -0,0 +1,95 @@
+import { taggedTemplateHelpers as utils } from '@cli/utils';
+
+const drupalLayout = `
+const navItems = [
+    { linkText: '🏠 Home', href: '/' },
+    { linkText: '📰 Articles', href: '/articles' },
+    { linkText: '📑 Pages', href: '/pages' },
+    { linkText: '⚛️ Examples', href: '/examples' },
+];
+const footerMenuItems = footerMenu?.map(({ title, url, parent }) => ({
+    linkText: title,
+    href: url,
+    parent: parent,
+}));
+`;
+const wpLayout = `
+	const navItems = [
+		{
+			linkText: '🏠 Home',
+			href: '/',
+		},
+		{
+			linkText: '📰 Posts',
+			href: '/posts',
+		},
+		{
+			linkText: '📑 Pages',
+			href: '/pages',
+		},
+		{
+			linkText: '⚛️ Examples',
+			href: '/examples',
+		},
+	];
+
+	const footerMenuItems = footerMenu?.map(({ path, label }) => ({
+		linkText: label,
+		href: path,
+		parent: null,
+	}));
+`;
+
+const wpSig = `		<a
+						className="text-white hover:text-blue-100 underline"
+						href="https://nextjs.org/"
+					>
+						Next.js
+					</a>{' '}
+					and{' '}
+					<a  
+						className="text-blue-500 underline hover:text-blue-100"
+						href="https://wordpress.com/"
+					>
+						WordPress
+					</a>`;
+const drupalSig = `<a
+						href="https://nextjs.org/"
+					>
+						Next.js
+					</a>{' '}
+					and{' '}
+					<a
+						href="https://www.drupal.org/"
+					>
+						Drupal
+					</a>`;
+
+export const layoutTemplate = (
+	search: boolean,
+	cmsType: string,
+) => /* jsx */ `import { Footer, Header, PreviewRibbon } from '@pantheon-systems/nextjs-kit';
```

Comment:
Use named args if there is more than one, this way it's easier to tell what these args mean when we're calling the templates
```suggestion
export const layoutTemplate = ({ search, cmsType } :{
	search: boolean,
	cmsType: string,
}) => /* jsx */ `import { Footer, Header, PreviewRibbon } from '@pantheon-systems/nextjs-kit';
```